pkg/machine/qemu/machine.go

```diff
@@ -163,9 +162,13 @@ func (v *MachineVM) Init(opts machine.InitOptions) error {
 	// Add location of bootable image
 	v.CmdLine = append(v.CmdLine, "-drive", "if=virtio,file="+v.ImagePath)
 	// This kind of stinks but no other way around this r/n
-	uri := machine.SSHRemoteConnection.MakeSSHURL("localhost", "/run/user/1000/podman/podman.sock", strconv.Itoa(v.Port), v.RemoteUsername)
-	if err := machine.AddConnection(&uri, v.Name, filepath.Join(sshDir, v.Name), opts.IsDefault); err != nil {
-		return err
+	if len(opts.IgnitionPath) < 1 {
+		uri := machine.SSHRemoteConnection.MakeSSHURL("localhost", "/run/user/1000/podman/podman.sock", strconv.Itoa(v.Port), v.RemoteUsername)
+		if err := machine.AddConnection(&uri, v.Name, filepath.Join(sshDir, v.Name), opts.IsDefault); err != nil {
+			return err
+		}
+	} else {
+		fmt.Println("An ignition path was provided.  No SSH connection was added to Podman")
```

Comment:
Is this necessary, or just debugging information?

Reply:
i think it is needed ... esp for those who do not read docs.